sql/core/src/test/resources/sql-tests/inputs/time.sql

```diff
@@ -82,3 +82,11 @@ SELECT '23:59:59.999999' :: TIME - INTERVAL '23:59:59.999999' HOUR TO SECOND;
 SELECT '00:00:00.0001' :: TIME(4) - INTERVAL '0 00:00:00.0001' DAY TO SECOND;
 SELECT '08:30' :: TIME(0) - INTERVAL '6' HOUR;
 SELECT '10:00:01' :: TIME(1) - INTERVAL '1' MONTH;
+
+-- Subtract times
+SELECT TIME'12:30:41' - TIME'10:00';
+SELECT TIME'08:30' - NULL;
+SELECT NULL - TIME'10:32';
+SELECT TIME'12:30:41.123' - TIMESTAMP'10:00:01';
+SELECT '12:30:41.123' - TIME'10:00:01';
```

Comment:
An example where start > end perhaps? Showing negative results.

Reply:
+1

Reply:
oh do we support it? the hour range is 0-23 for TIME type, right?

Reply:
added such check, thx